proc_linewidth.py

```diff
@@ -72,23 +86,32 @@
     print(type(result),result.shape)
     guess_nddata = nddata(result, [-1], ['$B_0$']).setaxis(
             '$B_0$',x_finer).set_units('$B_0$',d.get_units('$B_0$'))
-    fl.plot(d, label='data')
-    fl.plot(guess_nddata,':', label='guess')
-    fl.next('guess')
+    plot(d, label='data')
+    plot(guess_nddata,':', label='guess')
     model = s.Model({y_var:dVoigt})
-    guess = InteractiveGuess(model, y=d.data.real, B=d.getaxis('$B_0$'), n_points=500)
-    guess.execute()
-    print(guess)
+    if interactive:
+        guess = InteractiveGuess(model, y=d.data.real, B=d.getaxis('$B_0$'), n_points=500)
+        guess.execute()
+        print(guess)
     y_var = s.Variable('y')
     print("about to run fit")
-    fit = s.Fit(model, d.getaxis('$B_0$'), d.data.real)
+    fit = s.Fit(model, d.getaxis('$B_0$'), d.data.real)#, minimizer=MINPACK) # really want to use minpack here, but gives "not proper array of floats
     fit_result = fit.execute()
-    fl.next('data with fit')
+    print("fit is done")
+    plt.figure()
+    plt.title('data with fit')
     plot(d, '.', label='data')
     fit_nddata = nddata(
             fit.model(B=x_finer, **fit_result.params).y,
             [-1], ['$B_0$']).setaxis('$B_0$', x_finer)
-    fl.plot(fit_nddata, label='fit')
+    plot(fit_nddata, label='fit')
     print(fit_result)
-
-fl.show()
+    conc_list.append(concentration)
+    R_list.append(fit_result.params['R'])
+    sigma_list.append(fit_result.params['sigma'])
+plt.figure()
+plot(conc_list,R_list,'x',label='R')
+plot(conc_list,sigma_list,'x',label=r'$\sigma$')
+plt.legend(**dict(bbox_to_anchor=(1.05,1), loc=2, borderaxespad=0.))
+plt.ylabel('concentration')
+plt.show()
```

Comment:
And so for this PR you want me to try to integrate in the global fit?

Reply:
No, here I am making sure that you understand everything that we did, since this was from the earlier meeting. The global fit should go in a separate pull request.

Reply:
okay so the only thing that is a little wonky to me is the interactive guess but I can clarify this by going to the documentation. Other than that this should be good to merge!